ethclient/ethclient.go

```diff
@@ -108,10 +108,10 @@ func (ec *Client) getBlock(ctx context.Context, method string, args ...interface
 	if head.UncleHash != types.EmptyUncleHash && len(body.UncleHashes) == 0 {
 		return nil, fmt.Errorf("server returned empty uncle list but block header indicates uncles")
 	}
-	if head.TxHash == types.EmptyRootHash && len(body.Transactions) > 0 {
+	if head.TxHash == types.EmptyBMTRootHash && len(body.Transactions) > 0 {
```

Comment:
`ethclient` is ethereum JSONRPC client and it is used when to connect to root chain. On the other hand, `plsclient` is used to connect plasma JSONRPC server.

It have to use `EmptyRootHash` instead of `EmptyBMTRootHash` because ethereum client use merkle patricia trie.